raiden/transfer/mediated_transfer/events.py

```diff
@@ -40,6 +55,33 @@ def __init__(
         self.receiver = receiver
 
 
+class SendMediatedTransfer2(Event):
```

Comment:
I started writting a commit to give more proper names to these but I am not sure if this makes sense.

Are the `2` names only temporary so that both old and new events can live alongside each other for the moment?

I am talking about those 3:

```
Rename SendMediatedTransfer2
Rename SendBalanceProof2
Rename SendRefundTransfer2
```

Especially the `SendRefundTransfer2` seems to just be the new version of `SendRefundTransfer` and not something serving a different purpose.

Reply:
> Are the 2 names only temporary

Yes, they were introduced for the same reason as https://github.com/raiden-network/raiden/pull/1270#issue-170488232

Reply:
So yeah then no need to even bother with renaming at this point.